### PR TITLE
Multiple big improvements

### PR DESCRIPTION
**Pull request by Joolee**
### Changes
* Fixed Wake on Lan feature (Allows to turn the TV on when it has been off for >10 minutes)
* Made app ~75% smaller by removing or substituting big dependencies (Makes development faster, reduces memory (by ~33%) and CPU usage and greatly reduces possible vulnerabilities)
* Added English/Dutch translations for remote button names (in the action card)
* Stop polling the TV when you remove it from Homey (Might fix some crashes)
* Split notifyChange and device polling into separate threads (should stop on/off flickering)

### CODE REVIEW

**Comment by lucasvdh** [comment]
Hi @Joolee,

Thank you for this big contribution! 🙌

I've not been able to really give this app as much love as I would've liked because I'm still struggling with RSI. I will have a look at this pull request as soon as I'm back from vacation (in about a week).

**Comment by Joolee** [comment]
Enjoy your vacation :)

File: package.json [hunk 1]
```diff
@@ -3,13 +3,8 @@
   "name": "philips-jointspace",
   "version": "0.5.1",
   "dependencies": {
-    "crypto": "^1.0.1",
```
Comment (lucasvdh) [inline package.json:6]: The `crypto` dependency is still used during pairing but no longer referenced in the `package.json`.

**Review by lucasvdh — COMMENTED** [review]
@Joolee overall this is looking very good, but I'd like to clarify my comment below. 

Was this a mistake or an intentional change for a reason that I'm missing?

File: package.json (hunk 1 above)
Comment (lucasvdh) [reply package.json:6]: Thanks for clearing things up!

**Review by lucasvdh — APPROVED** [review]

**Pull request merged**